opimodel/widgets.py

```diff
@@ -232,6 +232,8 @@ class GroupingContainer(Widget):
     def __init__(self, x, y, width, height):
         super(GroupingContainer, self).__init__(
             GroupingContainer.TYPE_ID, x, y, width, height)
+        self.lock_children = True
```

Comment:
why lock_children be default; this is an annoying setting if you're trying to manually update panels and is invisible to anyone running them.